src/fu-udev-backend.c

```diff
@@ -605,6 +605,10 @@ fu_udev_backend_netlink_cb(gint fd, GIOCondition condition, gpointer user_data)
 			g_debug("ignoring netlink message: %s", error_local->message);
 			return TRUE;
 		}
+		if (g_error_matches(error_local, FWUPD_ERROR, FWUPD_ERROR_NOT_FOUND)) {
+			g_debug("ignoring netlink message: %s", error_local->message);
+			return TRUE;
+		}
```

Comment:
do you want to squash the two conditionals together?

Reply:
sure